README.md

```diff
@@ -455,11 +456,29 @@ Dodge et. al 2015. Tests Memory Networks on 4 tasks including reddit dialog task
 
 * [Hindi Dependency Treebank](http://ltrc.iiit.ac.in/treebank_H2014/) - A multi-representational multi-layered treebank for Hindi and Urdu
 * [Universal Dependencies Treebank in Hindi](http://universaldependencies.org/treebanks/hi/index.html)
-     * [Parallel Universal Dependencies Treebank in Hindi](http://universaldependencies.org/treebanks/hi_pud/index.html) - A smaller part of the above-mentioned treebank.
+     * [Parallel Universal Dependencies Treebank in Hindi](http://universaldependencies.org/treebanks/hi_pud/index.html) - A smaller 
+     part of the above-mentioned treebank.
+     
+## NLP in Thai
+
+[Back to Top](#contents)
+
+### Libraries
+
+* [PyThaiNLP](https://github.com/wannaphongcom/pythainlp) - Thai NLP in Python Package
+* [JTCC](https://github.com/wittawatj/jtcc)- A character cluster library in Java
+* [SWATH](http://www.cs.cmu.edu/~paisarn/software.html) - Word segmentation in C language
```

Comment:
SWATH is too old (2003) and no longer maintained prima facie. Please do not include here.